src/table/components/TableBodyWrapper.jsx

```diff
@@ -18,9 +18,11 @@ function TableBodyWrapper({
   setShouldRefocus,
   keyboard,
   tableWrapperRef,
+  tableContainerRef,
   announce,
 }) {
   const { rows, columns, paginationNeeded } = tableData;
+  const focusedCellCoord = useContextSelector(TableContext, (value) => value.focusedCellCoord);
```

Comment:
Unfortunately this will break the performance improvement I did. `focusCellCoord` updates everytime you click to select, thus re-rendering the body component and all the cells.

Reply:
updated to another way of putting focusedCellCoord and logic inside tablewrapper.